docusaurus/docs/introduction/plugin-lifecycle.md

```diff
@@ -0,0 +1,89 @@
+---
+id: plugin-lifecycle
+title: Life cycle of a plugin
+description: Learn about the life cycle of a Grafana plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - lifecycle
+  - life cycle
+  - loading
+  - unloading
+  - installation
+sidebar_position: 2.5
+---
+
+# Life cycle of a plugin
+
+This document describes the various phases of a plugin such as installation and loading. We will describe the differences in a plugin's life cycle depending on its type and whether or not it has a [backend](backend.md).  
+
+:::note
+
+The life cycle of plugins is tracked in-memory and is not persisted in Grafana’s database. 
```

Comment:
I think it would be good to explain why this matters to the reader. I _believe_ it means that it happens very time the grafana server is restarted. So something like "This means that the phases described below occur every time the server is restarted." Someone on the team could confirm/amend this.

If this indeed the case, it might be better to move this note to the Loading plugins section.

Reply:
done. @marefr - can you add to/improve on david's suggestion?